internal/commands/login/inputs.go

```diff
@@ -66,19 +71,23 @@ func (i *inputs) Resolve(profile *user.Profile, ui terminal.UI) error {
 func (i *inputs) resolveCloudCredentials(u user.Credentials) []*survey.Question {
 	var questions []*survey.Question
 
-	if i.PublicAPIKey == "" {
-		if u.PublicAPIKey == "" {
+	if i.PublicAPIKey == "" || i.ForceBrowser {
```

Comment:
spent a bit of time thinking about this and I could very well be missing some aspect of the desired logic here, but it _seems_ to me that `ForceBrowser` could simply be its own, static set of questions.

if we switched the `select` above to not be based purely off `i.AuthType` but rather just a generic one:

```go
	switch {
	case i.ForceBrowser:
		questions = []*survey.Question{
			&survey.Question{
				Name:   inputFieldPublicAPIKey,
				Prompt: &survey.Input{Message: "Public API Key", Default: u.PublicAPIKey},
			},
			&survey.Question{
				Name:   inputFieldPrivateAPIKey,
				Prompt: &survey.Password{Message: "Private API Key"},
			},
		}
	case i.AuthType == authTypeCloud:
		questions = i.resolveCloudCredentials(u)
	case i.AuthType == authTypeLocal:
		questions = i.resolveLocalCredentials(u)
	default: // ...
    }
```

then we don't need any extra wrinkles in the `resolveCloudCredentials` impl